app/controllers/api/v1/accounts/relationships_controller.rb

```diff
@@ -14,6 +14,21 @@ def index
 
   private
 
+  def all_accounts
+    Account.where(id: account_ids).select('id')
+  end
```

Comment:
This is redundant, `account_ids` is a list of `accounts` `id` property, which is then used to in a query to find all accounts by those ids, just to select the id.

Reply:
Written plainly: It's like I gave you a bunch of apples and then asked you for all of the apples I gave you. I should just keep the apples.

Reply:
That said there is a material difference between `all_accounts` and `account_ids` and that's a list of `Account` objects with an `id` property value. It would be much faster to simply map the `account_ids` values into an `Account` object.

Reply:
It's entirely possible that https://github.com/mastodon/mastodon/pull/23455/files#diff-31a7375ea1b1a11051b2a973de9114d94b96097378c46513f6e14c388cc4a86cR12 requires a real `Account` but that should be fixed. I don't think anything is truly wrong with this.

Reply:
That's understandable. I kept the query and the select here because that was the original code. Also, what do you think about naming this method `account_ids` instead of `all_accounts`?